src/v/cloud_storage/tests/remote_file_test.cc

```diff
@@ -87,7 +86,11 @@ class remote_file_fixture
         ss::file f = ss::open_file_dma(file_path.string(), flags).get();
         return remote.local()
           .upload_controller_snapshot(
-            bucket, remote_path, f, retry_node, always_continue)
+            cloud_storage_clients::bucket_name{bucket_name},
```

Comment:
it feels like this could be less verbose if `bucket_name` will have the right type in the fixture

Reply:
nit

Reply:
Fixed.